airqo-platform/AirQo-frontend
# [Website] Highlights Endpoint Integration

#### Summary of Changes (What does this PR do?)
- Features integration of highlights endpoint to serve the highlights section data in frontend. Also includes new loadspinners for section loading and image loading.

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
- git checkout staging
- git pull origin staging
- git checkout origin _ft-highlights-integration_ 
- To make changes, make sure to have your _postgres_ database set up and _URI_ updated in your local _.env_ file. Also update the following ENV variable: _REACT_APP_WEBSITE_BASE_URL=http://localhost:8000/api/v1_ 
- Finally, run the _serve_ script and _django server_  to add data through the admin portal and see it served in the frontend. [Instructions](https://github.com/airqo-platform/AirQo-frontend/edit/staging/website/README.md#running-the-website-application)

#### What are the relevant tickets?
- [WEB-295]

#### Screenshots
Admin Portal
![Screenshot from 2022-08-10 20-41-19](https://user-images.githubusercontent.com/53012890/183983879-583b1b9e-258e-4885-87b8-b9077ec776cf.png)

Frontend
![Screenshot from 2022-08-10 20-41-25](https://user-images.githubusercontent.com/53012890/183983898-1af063d1-e66e-4d22-9f66-9737780d8814.png)



[WEB-295]: https://airqoteam.atlassian.net/browse/WEB-295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
